admin/server/alerts.go

```diff
@@ -35,10 +35,7 @@ func (s *Server) GetAlertMeta(ctx context.Context, req *adminv1.GetAlertMetaRequ
 
 	proj, err := s.admin.DB.FindProject(ctx, req.ProjectId)
 	if err != nil {
-		if errors.Is(err, database.ErrNotFound) {
-			return nil, status.Error(codes.NotFound, "project not found")
-		}
-		return nil, status.Error(codes.InvalidArgument, err.Error())
```

Comment:
not sure what was the purpose of using InvalidArgument here but now Internal error would be returned which is more appropriate I guess. There are many instances like this below.

Reply:
Yeah `Internal` is more appropriate here. There were many such cases of inconsistent error handling, hopefully there will be fewer now.